app/src/org/commcare/dalvik/activities/LoginActivity.java

```diff
@@ -571,12 +556,21 @@ public CustomProgressDialog generateProgressDialog(int taskId) {
         }
         return dialog;
     }
-    
+
     @Override
     public boolean isBackEnabled() {
-        return false;
+        return true;

```

Comment:
All this does is make the "<" chevron appear next to the Activity name, btw, which I don't think was anticipated.